net/net.sh

```diff
@@ -145,6 +145,9 @@ Operate a configured testnet
  startnode/stopnode-specific options:
    -i [ip address]                    - IP Address of the node to start or stop
 
+ startnode specific option:
+   --wen-restart                      - Apply wen restart config during startNode.
```

Comment:
this flag takes an argument? not clear from this help message

Reply:
Changed.